docpage/docs/Users.md

```diff
@@ -40,11 +40,11 @@ python dcnios-cli.py changeSchedule --host={nifi-endpoint} \
 
 ## File workflow configuration structure (Yaml structure)
 
-Here, we will explain the workflow definition, the structure of the configuration file, and the information the user has to know about each third-party connection. DCNiOS deploys and configures all the definitions in Apache NiFi.
+Here, we explain the workflow definition, the structure of the configuration file, and the information the user has to know about each third-party connection. DCNiOS deploys and configures all the definitions in Apache NiFi.
```

Comment:
Here, we describe the workflow definition, the structure of the configuration file, and the necessary information for users regarding each third-party connection. DCNiOS handles the deployment and configuration of all definitions in Apache NiFi.